src/components/Skeleton/Skeleton.tsx

```diff
@@ -0,0 +1,61 @@
+import * as React from 'react';
+import { classNames } from '../../utils/classNames';
+
+// Animation types for the skeleton component.
+export type SkeletonAnimation = 'pulse'
+    | 'wave'
+    | false;
+
+// Skeleton variant types. 
+export type SkeletonVariant = 'circular'
+    | 'rectangular'
+    | 'rounded'
+    | 'text'
+    | string;
```

Comment:
this means any string value would be valid for variants props -- may lead to misuse of variant prop? - what if one provided an invalid values that do not match with any specified variants (? - just my perspective)